src/routes/controllers/users.controller.js

```diff
@@ -40,13 +40,41 @@ export async function getAuthorProfile(req, res) {
       where: {
         username,
       },
+      include: {
+        model: Article,
+        as: 'publications',
+      },
     });
     if (!authorProfile) {
       return errorResponse(res, 404, 'Author not found');
     }
-    return successResponse(res, 200, 'Get profile request successful', {
-      authorProfile,
-    });
+
+    const fullProfile = authorProfile.toJSON();
+    fullProfile.followers = await authorProfile
+      .getFollowers()
+      .map(followee => ({
+        id: followee.id,
```

Comment:
Is this endpoint returning `followers`, `followings`, `articles`? If yes, why not just attach it to the single query for getting the user profile. i.e. include `['followings', 'followers', 'publications']`?